ui/shared/utils/constants.js

```diff
@@ -111,7 +111,7 @@ export const MATCHMAKER_CONTACT_URL_FIELD = {
 
 // SAMPLES
 
-export const DATASET_TYPE_VARIANT_CALLS = 'VARIANTS'
+export const DATASET_TYPE_VARIANT_CALLS = 'SNV_INDEL'
```

Comment:
Is it painful to also rename the constant `DATASET_TYPE_SNV_INDEL_CALLS`?

Reply:
nope! thats a good idea